portals/developer-portal/samples/layouts/green-theme/styles/main.css

```
@@ -0,0 +1,507 @@
+/* --SubTitle-- */
+.api-md-content h4,
+h5,
+h6 {
+  font-size: 20px;
+  color: var(--main-text-color);
+}
+
+/* --Paragraph-- */
+.api-md-content p {
+  font-size: 15px;
+  color: var (--main-text-color);
```

Stylelint (17.12.0)

[error] Missing var function for "--main-text-color" (custom-property-no-missing-var-function)

(custom-property-no-missing-var-function)